### DISCUSSION TITLE
Evolution on Football Players Tracking

### DISCUSSION THREAD

**Discussion by massimo1108** [roboflow/notebooks#62] [Q&A]
I've several question about your videos:

1) the model request to have the same perspective. It means any time extra manual work is needed to fine-tune the identification
2) How difficult should be tracking on CSV speed or acceleration for players and ball? Would it be tricky defining the start and stop of a sudden change in speed? Only peak speed or number of runs above its average is possible.
3) Distance between players could be a fascinating perspective to be tracked, probably more accessible than speed. What do you think?

M

**Comment by SkalskiP** [comment]
Hi, @massimo1108 👋🏻!

> the model request to have the same perspective. It means any time extra manual work is needed to fine-tune the identification

Could you elaborate? I don't understand that question. 

> How difficult should be tracking on CSV speed or acceleration for players and ball? Would it be tricky defining the start and stop of a sudden change in speed? Only peak speed or number of runs above its average is possible.

The first big problem you must solve is converting the camera perspective into a bird's eye view. Without that, you can't calculate any speed. 

> Distance between players could be a fascinating perspective to be tracked, probably more accessible than speed. What do you think?

Same thing as above. We first need to be able to get BEV.
